### PR TITLE
Add Lippupiste ticket system

### PR DESCRIPTION
**Pull request by nikomakela**
Related to https://github.com/City-of-Helsinki/kukkuu-admin/pull/234

KK-1005. Support for Lippupiste events. They work exactly the same as Ticketmaster events. The texts related to the Ticketmaster replaced with more generic texts so the Ticketmaster words are replaced with "external ticket system".

image

image

image

image

### CODE REVIEW

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr470.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kukkuu-ui-pr470.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr470.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kukkuu-ui-pr470.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/event/ExternalTicketSystemEventIsEnrolled.tsx [hunk 1]
```diff
@@ -2,28 +2,28 @@ import { useTranslation } from 'react-i18next';
...
-import { ticketmasterEventQuery } from './queries/ticketmasterEventQuery';
+import styles from './externalTicketSystemEventIsEnrolled.module.scss';
+import { externalTicketSystemEventQuery } from './queries/externalTicketSystemEventQuery';
 import {
-  ticketmasterEventQuery as TicketmasterEventQueryType,
+  externalTicketSystemEventQuery as ExternalTicketSystemEventQueryType,
   // eslint-disable-next-line max-len
-  ticketmasterEventQuery_event_ticketSystem_TicketmasterEventTicketSystem as EventTicketSystem,
-} from '../api/generatedTypes/ticketmasterEventQuery';
+  externalTicketSystemEventQuery_event_ticketSystem_TicketmasterEventTicketSystem as EventTicketSystem,
```
Comment (karisal-anders) [inline src/domain/event/ExternalTicketSystemEventIsEnrolled.tsx:10]: Doesn't this assume that `externalTicketSystemEventQuery_event_ticketSystem_TicketmasterEventTicketSystem` is the same interface as `externalTicketSystemEventQuery_event_ticketSystem_LippupisteEventTicketSystem` for this file's code to work correctly? If so, then should this dependency perhaps be made explicit by combining `externalTicketSystemEventQuery_event_ticketSystem_TicketmasterEventTicketSystem` and `externalTicketSystemEventQuery_event_ticketSystem_LippupisteEventTicketSystem` into one or some other way?

**Comment by nikomakela** [comment]
The SonarCloud Quality Gate raises up a warning about the code coverage, but I think it calculates the generated code part of the changes as well. Also the test coverage is not very good overall at all. We need to write several tests here, not only related to the Lippupiste, but related to everything. Especially the external ticket system does not have many tests (or does it have 2 now?).

**Review by karisal-anders — APPROVED** [review]
Left one comment about implicitly assuming two types are compatible, it would be good to fix / make explicit IMHO.

The only `.kuva` stuff left are AFAIK:
 - https://kukkuu-ui.test.kuva.hel.ninja/ -> https://kukkuu-ui.test.hel.ninja/ in the README.md
- https://tunnistamo.test.kuva.hel.ninja -> https://tunnistamo.test.hel.ninja in the .env.* files
  - I hear your tunnistamo-related [answer](https://github.com/City-of-Helsinki/kukkuu-admin/pull/234#issuecomment-1540213556) but wanted to say this here also as it applies.

File: src/domain/event/ExternalTicketSystemEventIsEnrolled.tsx (hunk 1 above)
Comment (nikomakela) [reply src/domain/event/ExternalTicketSystemEventIsEnrolled.tsx:10]: well, they are the same, but it definitely needs to point some where else, in an union type like you said. I'll fix this.

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr470.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kukkuu-ui-pr470.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**